src/test/resources/sftp/local/OutBound/status-20221101171156056.json

```diff
@@ -0,0 +1 @@
+Hello World 1
```

Comment:
Is it worth making these files valid json, and conforming to `PrintResponses` ?
Otherwise when we get to the next part (processing the files downloaded) the tests will be unhappy

Reply:
It's not in the scope of this PR to see how message/files look like, so will make this change in next PR when listener actually reads it

Reply:
OK